index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title> PRELOG </title>
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <script src="https://use.fontawesome.com/23bea62652.js"></script>
+  </head>
+  <body>
+    
+    <!-- start the Header section of the website-->
+    <nav>
+    <header >
+        <img class="logo" src="./images/logo (1).png" width="190px" height="43.7" alt="logo" />
+      
+        <ul class="items">
+          <li id="benefit-item"><a href="./pagess/benefit.html">Benefits </a></li>
+          <li id="blog-item"><a href="./pagess/blogs.html">Blogs </a></li>
+          <li id="work-item"><a href="./pagess/work.html">How its work</a> </li>
+          <li id="login-item"><a href="./pagess/login.html">Login</a></li>
+          <li id="resister-item"><a href="./pagess/resister.html">Resister</a></li>
+        </ul>
+      
+            <input
+              type="text"
+              placeholder="search"
+              name="search"
+            />
+            <div class="menu-icon">
+            <i class="fa fa-bars "  aria-hidden="true"></i>
+          </div>
+       
+    </header>
+  </nav>
+    <!--Adding the Home page Picture -->
+    <section class="landing-container">
+      <img src="./images/nature bubbles.png" alt="AMAZING NATURE" class="landing-img" />
+      <div class="content">
+        <h2>
+          This is Place of <br />
+          <span> Secure. & Peace.</span>
+        </h2>
+        <p>get your things secure and take your mind in peace!!!</p>
+      </div>
+    </section>
+    <!--Adding the footer section-->
+    <footer>
+      <div>
+        <p class="copy-right">
+          Copyright © 2045 <a href="#">Your Site</a> , All rights reserved. |
```

Comment:
you can get Copywrite icon `©` in HTML instead of copy it by writing `&copy`.

` <p class="copy-right">
          Copyright &copy 2045 <a href="#">Your Site</a> , All rights reserved. |
          Designed By :<a href="#">IndioWeb</a>
        </p>`